File: java/src/test/java/LoginPageTest.java
import org.junit.Assert;
import org.junit.Test;
import org.openqa.selenium.By;

public class LoginPageTest extends TestCase {

    @Test
    public void loginToSwagLabs() {
        driver.get("https://www.saucedemo.com/v1/");

        driver.findElement(By.id("user-name")).sendKeys("standard_user");
        driver.findElement(By.id("password")).sendKeys("secret_sauce");
        driver.findElement(By.id("login-button")).click();

        Assert.assertTrue(driver.getCurrentUrl().contains("inventory"));
    }

    @Test
    public void invalidLogin() {
        driver.get("https://www.saucedemo.com/v1/");

        driver.findElement(By.id("user-name")).sendKeys("bad");
        driver.findElement(By.id("password")).sendKeys("bad");
        driver.findElement(By.id("login-button")).click();

        Assert.assertTrue(driver.findElement(By.cssSelector(".error-button")).isDisplayed());
    }
}

File: kotlin/src/test/kotlin/LoginPageTest.kt
import org.junit.jupiter.api.*
import org.openqa.selenium.By


@TestInstance(TestInstance.Lifecycle.PER_CLASS)
class LoginPageTest() : TestCase() {

    @Test
    fun loginToSwagLabs() {
        driver.get("https://www.saucedemo.com/v1/")

        driver.findElement(By.id("user-name")).sendKeys("standard_user")
        driver.findElement(By.id("password")).sendKeys("secret_sauce")
        driver.findElement(By.id("login-button")).click()

        Assertions.assertTrue("inventory" in driver.currentUrl, "Did not log into Swag Labs")
    }

    @Test
    fun invalidLogin() {
        driver.get("https://www.saucedemo.com/v1/")

        driver.findElement(By.id("user-name")).sendKeys("bad")
        driver.findElement(By.id("password")).sendKeys("bad")
        driver.findElement(By.id("login-button")).click()

        Assertions.assertTrue(driver.findElement(By.cssSelector(".error-button")).isDisplayed)
    }


}